src/chains/migrations/0019_add_safe_apps_rpc.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.5 on 2021-07-22 09:49
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("chains", "0018_chain_rpc_authentication"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="chain",
+            name="safe_apps_rpc_authentication",
+            field=models.CharField(
+                choices=[
+                    ("API_KEY_PATH", "Api Key Path"),
+                    ("NO_AUTHENTICATION", "No Authentication"),
+                ],
+                default="NO_AUTHENTICATION",
+                max_length=255,
+            ),
+        ),
+        migrations.AddField(
+            model_name="chain",
+            name="safe_apps_rpc_uri",
+            field=models.URLField(default=None),
```

Comment:
The default value here is `None` which breaks the non-nullability constraints set by the model 🤔

Reply:
Maybe we can set the default to the value of `rpc_uri`. The same for authentication